spec/disposable-spec.coffee

```diff
@@ -20,3 +20,13 @@ describe "Disposable", ->
 
     disposable.dispose()
     expect(disposalAction.callCount).toBe 1
+
+  describe "::isDisposable(object)", ->
```

Comment:
Sounds good, I had a look at [how we did it in text-buffer](https://github.com/atom/text-buffer/blob/master/spec/point-spec.coffee#L11) and I assumed it was a standard to refer to every kind of method.

Thanks for pointing this out! :+1: